components/camel-rabbitmq/src/test/java/org/apache/camel/component/rabbitmq/RabbitMQSupendResumeIntTest.java

```diff
@@ -28,12 +28,12 @@
 import org.junit.Test;
 
 public class RabbitMQSupendResumeIntTest extends CamelTestSupport {
-    private static final String EXCHANGE = "ex4";
+    private static final String EXCHANGE = "ex6";
```

Comment:
This used the same exchange and queue as another test, but it used different settings so declaring the exchange and queue would randomly fail when running all the tests.